installer/Templates/Head.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+
+namespace Installer\Templates;
+
+
+use Template;
+
+class Head implements Template
+{
+
+	/**
+	 * @var array list of the steps (ordered)
+	 */
+	private $steps = ['Welcome', 'Requirements', 'Permissions', 'Env', 'Migrate'];
+
+	private function get_index($step) {
+		return array_search($step , $this->steps);
+	}
+
+	public function print(array $input = [])
+	{
+		echo '<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Lychee Installer</title>
+        <link rel="icon" type="image/png" href="installer/assets/img/favicon/favicon-16x16.png" sizes="16x16"/>
+        <link rel="icon" type="image/png" href="installer/assets/img/favicon/favicon-32x32.png" sizes="32x32"/>
+        <link rel="icon" type="image/png" href="installer/assets/img/favicon/favicon-96x96.png" sizes="96x96"/>
+        <link href="installer/assets/css/style.min.css" rel="stylesheet"/>
+        
+        <!-- @yield(\'style\') -->
+    </head>
+    <body>
+        <div class="master">
+            <div class="box">
+                <div class="header">
+                    <h1 class="header__title">'.$input['title'].'</h1>
+                </div>
+                <ul class="step">
+                    <li class="step__divider"></li>
+                    <li class="step__item '.($input['step'] == 'Migrate' ? 'active' : '')
+			.'">';
+		echo '<i class="step__icon fa fa-server" aria-hidden="true"></i>';
+		echo '
+                    </li>';
+		echo '
+                    <li class="step__divider"></li>
+                    <li class="step__item '.($input['step'] == 'Env' ? 'active' : '')
+			.'">';
+		if ($this->get_index($input['step']) >= 3) {
+			echo '<a href="?step=env">
```

Comment:
Can you add a `title` or some such to these buttons so that there's an explanation when one hovers over? I don't find the icons to be particularly obvious :smile:.

Reply:
fixed.